challengeMk2/Views/StarSystemsPage.xaml.cs

```diff
@@ -31,6 +31,8 @@ protected override void OnAppearing()
 
             var vm = BindingContext as StarSystemsViewModel;
 
+            SystemCollection.SelectedItem = null;
```

Comment:
By the way, selectedItem should be set to null in the setter.

Reply:
It is set in the setter now and allows to reselect the same item without keep the selection when go back in the list view.